packages/sign_in_with_apple/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: sign_in_with_apple
-description: Flutter bridge to initiate Sign in with Apple (currently iOS only). Includes support for keychain entries as well as sign in with an Apple ID.
-version: 1.0.0
+description: Flutter bridge to initiate Sign in with Apple (currently iOS and macOS only). Includes support for keychain entries as well as signing in with an Apple ID.
+version: 1.1.0
 homepage: https://github.com/aboutyou/dart_packages/tree/master/packages/sign_in_with_apple
 repository: https://github.com/aboutyou/dart_packages
 
```

Comment:
@tp I think we should still have a flutter version constraint here

Reply:
Reverted, as it was the latest stable anyway and the one we tested with.